internal/services/recoveryservices/site_recovery_replicated_vm_resource.go

```diff
@@ -108,6 +108,24 @@ func resourceSiteRecoveryReplicatedVM() *pluginsdk.Resource {
 				Optional:         true,
 				ValidateFunc:     azure.ValidateResourceID,
 				DiffSuppressFunc: suppress.CaseDifference,
+				ConflictsWith: []string{
+					"target_availability_zone",
+				},
+			},
+			"target_availability_zone": {
+				Type:             pluginsdk.TypeString,
+				Optional:         true,
+				// Default:  "None",
+				Computed: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"1",
+					"2",
+					"3",
+				}, false),
+				ConflictsWith: []string{
+					"target_availability_set_id",
+				},
```

Comment:
As discussed in #14588 we're making Zones consistent, so this needs to become:

```suggestion
			"target_zone": commonschema.ZonesSingle()
```